Pool/ScoreTheme.vb

```diff
@@ -1,70 +1,45 @@
-﻿Imports System.Windows.Forms
+﻿Imports System.Text
+Imports System.Windows.Forms
 Imports Pool.Logging
 Public Class ScoreTheme
-#Region "Props"
-    Public Enum AppState
-        NoPlayerEx = -1
-        Start = 0
-        Register = 1
-        SelectPlayer = 2
-        Winner = 3
-        Switch = 4
-        Add = 5
-        edit = 6
-        Delete = 7
-    End Enum
-    Private _screen As AppState
-    Private _form As Form
-    Public Sub New()
 
-    End Sub
-    Public Sub New(f As Form)
-        Form = f
-    End Sub
-    Public Property Screen As AppState
-        Get
-            Return _screen
-        End Get
-        Set(value As AppState)
-            _screen = value
-        End Set
-    End Property
-    Public Property Form As Form
-        Get
-            Return _form
-        End Get
-        Set(value As Form)
-            _form = value
-        End Set
-    End Property
-#End Region
-
-#Region "Set theme"
-    Public Sub SetErrorLabel(ByRef errorText As Label)
+#Region "Set Controls"
+    ''' <summary>
+    ''' Set Error label and color
+    ''' </summary>
+    ''' <param name="errorText"></param>
+    Public Shared Sub SetErrorLabel(ByRef errorText As Label)
         If errorText.Text.Contains("Error") Then
             errorText.ForeColor = Color.Red
             errorText.Visible = True
         Else
             errorText.Visible = False
-            errorText.ForeColor = Color.Green
+            errorText.ForeColor = Color.Lime
         End If
```

Comment:
We've changed the color to aquamarine, in the case we are setting visibility off anyways. Remove this if we are only showing it on an error that are a Red color.

Reply:
This has been removed as we don't show this label unless there's an error.